.github/workflows/deploy.yml

```
@@ -140,6 +140,13 @@ jobs:
           NOD_CONTRACT_ADDRESS=$(./scripts/init_contract.sh "Nod" $NOD_CODE_ID "$INIT_PAYLOAD")
           echo "NOD_CONTRACT_ADDRESS=$NOD_CONTRACT_ADDRESS" >> $GITHUB_ENV
 
+      - name: Deploy Agent Registry
+        run: |
+          AGENT_REGISTRY_CODE_ID=$(./scripts/store_code.sh artifacts/agent_registry.wasm)
+          INIT_PAYLOAD=$(jq -n --from-file ./contracts/agent-registry/fixtures/init.json)
+          AGENT_REGISTRY_CONTRACT_ADDRESS=$(./scripts/init_contract.sh "Agent Registry" $AGENT_REGISTRY_CODE_ID "$INIT_PAYLOAD")
+          echo "AGENT_REGISTRY_CONTRACT_ADDRESS=$AGENT_REGISTRY_CODE_ID" >> $GITHUB_ENV
```

possibly should be like this
```suggestion
          echo "AGENT_REGISTRY_CONTRACT_ADDRESS=$AGENT_REGISTRY_CONTRACT_ADDRESS" >> $GITHUB_ENV
```